lib/devise/custom_failure.rb

```
@@ -0,0 +1,16 @@
+class Devise::CustomFailure < Devise::FailureApp
+  def respond
+    if request.format.json?
+      json_error_response
+    else
+      binding.pry
```

rubocop (1.69.1)

[warning] Remove debugger entry point `binding.pry`.

(Lint/Debugger)